src/components/collapse/collapse.svelte

```diff
@@ -47,6 +47,15 @@
   }
 
   .leoCollapse {
+    --summary-padding: var(
+      --leo-collapse-summary-padding,
```

Comment:
We call it a title in the props, but a summary here. Should they be the same?

Reply:
I'm a little conflicted about this - I think it should be `summary`, because conceptually we could have more than just the title in here (i.e. a little subtitle). That said, we don't support slotting content in here (maybe we should though?).

In favor of keeping this as summary:
- We optionally have an icon in here (which is not part of the title)
- We display the open/close indicator here (which is not part of the title).